releasenotes/notes/session_support-90c41a3c3f58a46b.yaml

```
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    Added support for using session. The syntax here is slightly different than that
+    in ``qiskit-ibm-runtime``, but the functionality is similar.
+    Here is an example using session as a context manager::
+    
+      with backend.open_session() as session:
+        job = backend.run(ReferenceCircuits.bell())
+    
```

```suggestion
    Here is an example using session as a context manager:
    
    .. code-block::
    
      with backend.open_session() as session:
        job = backend.run(ReferenceCircuits.bell())
    
```
For code snippets in the release notes I think they need to be prefaced with `.. code-block::`

---

I am fine with your suggestion. I must say I saw other examples in the release notes without `code-block`, so not sure if this is mandatory.